package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cors",
   "description": "Node.js CORS middleware",
-  "version": "2.8.5",
+  "version": "2.9.0",
```

Comment:
I think this version change is supposed to decided during release, not now. Please revert this change.

Reply:
done, thanks!